hvac/v1/__init__.py

```diff
@@ -1199,11 +1199,15 @@ def transit_sign_data(self, name, input_data, key_version=None, algorithm=None,
             params['context'] = context
         if prehashed is not None:
             params['prehashed'] = prehashed
+        if sign_algorithm is not None:
+            params['signature_algorithm'] = sign_algorithm
+        else:
+            params['signature_algorithm'] = "pss"
 
         return self._post(url, json=params).json()
 
     def transit_verify_signed_data(self, name, input_data, algorithm=None, signature=None, hmac=None, context=None,
-                                   prehashed=None, mount_point='transit'):
+                                   prehashed=None, mount_point='transit', sign_algorithm = None):
```

Comment:
nitpick: please remove the extra spaces around to be in line with PEP8.

Reply:
will do